opensearchpy/connection/http_requests.py

```diff
@@ -187,13 +191,15 @@ def perform_request(  # type: ignore
             "allow_redirects": allow_redirects,
         }
         send_kwargs.update(settings)
+        start = time.time()
```

Comment:
I think the start at line 179 is fine. Any reason otherwise?

Reply:
Hello @VachaShah, I am adding start timer just before sending request. I think we don't need  timer to start before prepared_request right?